File: eval/fixtures/legacy-directory-backed-asmx-commerce/src/Legacy/Orders/OrderFulfillmentService.cs
using System.Web.Services;

namespace Acme.Legacy.Orders;

[WebService(Name = "OrderFulfillmentService", Namespace = "http://legacy.example/orders")]
public class OrderFulfillmentService : WebService
{
  private readonly OrderFulfillmentManager _manager = new OrderFulfillmentManager();

  [WebMethod(MessageName = "RepriceFulfillment")]
  public string RepriceFulfillment(string orderId)
  {
    return _manager.RepriceBacklog(orderId);
  }
}

File: eval/fixtures/legacy-directory-backed-jaxws-commerce/src/main/java/com/acme/legacy/orders/OrderFulfillmentService.java
package com.acme.legacy.orders;
import javax.jws.WebMethod;
import javax.jws.WebService;

@WebService(serviceName = "OrderFulfillmentService")
public class OrderFulfillmentService {
  private final OrderFulfillmentManager orderFulfillmentManager = new OrderFulfillmentManager();

  @WebMethod(operationName = "RepriceFulfillment")
  public FulfillmentRepriceResult repriceFulfillment(String orderId) {
    return orderFulfillmentManager.repriceBacklog(orderId);
  }
}
